ambry-router/src/main/java/com.github.ambry.router/DeleteOperation.java

```diff
@@ -216,8 +216,8 @@ private void cleanupExpiredInflightRequests() {
       DeleteRequestInfo deleteRequestInfo = itr.next().getValue();
       if (time.milliseconds() - deleteRequestInfo.startTimeMs > routerConfig.routerRequestTimeoutMs) {
         itr.remove();
-        NonBlockingRouterMetrics.NodeLevelMetrics dataNodeBasedMetrics =
-            routerMetrics.getDataNodeBasedMetrics(deleteRequestInfo.replica.getDataNodeId());
+        responseHandler
```

Comment:
Two different changes. The line was removed as the variable was not getting used.